x-pack/docs/en/rest-api/security/create-service-token.asciidoc

```diff
@@ -0,0 +1,113 @@
+[role="xpack"]
+[[security-api-create-service-token]]
+=== Create service account token API
+
+beta::[]
+
+++++
+<titleabbrev>Create service account tokens</titleabbrev>
+++++
+
+Creates a service account token for access without requiring basic authentication.
+
+[[security-api-create-service-token-request]]
+==== {api-request-title}
+
+`POST /_security/service/<namespace>/<service>/credential/token/<name>`
+
+`PUT /_security/service/<namespace>/<service>/credential/token/<name>`
+
+`POST /_security/service/<namespace>/<service>/credential/token`
+
+[[security-api-create-service-token-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have at least the `manage_service_account` cluster privilege.
```

Comment:
@ywangd, do we need to add the `manage_service_account` cluster privileges to the list of [cluster privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html#privileges-list-cluster)? If so, would this description be accurate?
>Privileges to create, list, and delete service account tokens in any service account.

Reply:
Also, I got an error message when I sent a request indicating that "Security must be explicitly enabled". Should we also inform users that they need to set `xpack.security.enabled` to `true` in the `elasticsearch.yml` file as a prerequisite?

```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "exception",
        "reason" : "Security must be explicitly enabled when using a [basic] license. Enable security by setting [xpack.security.enabled] to [true] in the elasticsearch.yml file and restart the node."
      }
    ],
    "type" : "exception",
    "reason" : "Security must be explicitly enabled when using a [basic] license. Enable security by setting [xpack.security.enabled] to [true] in the elasticsearch.yml file and restart the node."
  },
  "status" : 500
}
```

Reply:
> do we need to add the `manage_service_account` cluster privileges to the list of [cluster privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html#privileges-list-cluster)? If so, would this description be accurate?
> 
> > Privileges to create, list, and delete service account tokens in any service account.

Good catch! Indeed, this should be added to the list. I updated it, but please feel free to modify as you see fit.

Reply:
> Also, I got an error message when I sent a request indicating that "Security must be explicitly enabled". Should we also inform users that they need to set `xpack.security.enabled` to `true` in the `elasticsearch.yml` file as a prerequisite?

These APIs are list under [Security APIs](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api.html). So I think enabling security is kinda a given in this context. If we do want to give extra hint to readers, maybe we could add it to the Security APIs page instead of every single page for each API?

Reply:
++ 
I'll add this information to the top of the Security APIs page.